app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/persona/AddFieldSheet.kt

```diff
@@ -46,27 +57,36 @@ fun AddFieldSheet(
     modifier: Modifier = Modifier,
     anyFieldSelected: Boolean
 ) {
-    Column(modifier = modifier) {
-        RadixCenteredTopAppBar(
-            title = stringResource(id = R.string.editPersona_addAField_title),
-            onBackClick = onBackClick,
-            contentColor = RadixTheme.colors.gray1,
+    Scaffold(modifier = modifier, topBar = {
+        Column {
+            RadixCenteredTopAppBar(
```

Comment:
Should it have an 'X' button instead of back? Maybe we should check with Aftab because the back icon doesn't make much sense on the bottom sheet here.

Reply:
Valid point. Waiting for the design for this, updated description